recipes/pykml/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pykml" %}
+{% set version = "0.1.0" %}
```

Comment:
No good reason... the Github release looks spurious. I updated to a new version (0.1.2) and made a GitHub and PyPI release, and updated the meta.yaml accordingly.